device/mellanox/x86_64-mlnx_msn4700-r0/Mellanox-SN4700-C128/pg_profile_lookup.ini

```diff
@@ -0,0 +1,41 @@
+##
```

Comment:
> the numbers are correct but we do not need the numbers for speed 200000 since there is no 200G port on this SKU.

customer asked us to add it. Maybe for future use.

Reply:
let's keep it then. 200G for 4x ports right?